blog/_posts/2023-09-06-client-quadlet.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+title: Run and expose Kubernetes Pod based Application using Quadlet and Inlets
+description: As a Kubernetes user, you can use the same YAML file to expose your application using Inlets
+author: Ygal Blum & Valentin Rothberg
+tags: podman quadlet containers
+author_img: ygal
```

Comment:
Please add your avatar, perhaps save your GitHub images images/ygal.png

Then run the blog locally using the instructions in the README file.

Check it looks correct on the roll-up page (see broken image)

![image](https://github.com/inlets/inlets.dev/assets/6358735/175abfbb-1d2d-4546-a243-16ec47588c73)

Reply:
Done (actually it looks for a `.jpg` file)